xr_sample_project/Source/xr_samplesproject/VRSample/VRCharacterController.h

```diff
@@ -124,6 +124,8 @@ class XR_SAMPLESPROJECT_API AVRCharacterController : public ACharacter
 	UMotionControllerComponent* rightMotionControllerInteractor;
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Variables", meta = (AllowPrivateAccess = "true"))
 	USpringArmComponent* springArmComponent;
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Variables", meta = (AllowPrivateAccess = "true"))
+	TEnumAsByte<EHMDTrackingOrigin::Type> TrackingOrigin;
```

Comment:
are we sure that the default value of this property for unreal <= 5.3 represents the same thing as the default value in unreal 5.4. if not, we will have a behavior difference between unreal versions

Reply:
Yes, I verified the sample still works and compiles in both 5.3 and 5.4 despite the default values being different. Making it a variable allows users to change the tracking origin type in the editor to keep the "low code - no code" abilities that our samples are intended to provide.

Reply:
yeah but if users don't change it manually, they will get different results in `=<5.3` and `5.4`

You could do this to address it

```suggestion
#if UE_VERSION_OLDER_THAN(5, 4, 0)
	TEnumAsByte<EHMDTrackingOrigin::Type> TrackingOrigin{EHMDTrackingOrigin::Floor};
#else
	TEnumAsByte<EHMDTrackingOrigin::Type> TrackingOrigin{EHMDTrackingOrigin::LocalFloor};
#endif
```

Reply:
When we update the project to 5.4, you won't be able to open any files, levels, blueprints, etc, in 5.3. So when we go through the process of changing the version the project uses, we will update the level to have this value be the equivalent of what it is in 5.3 (floor = local floor). I don't think we need to add this conditional for something that will be removed in ~ 3 months and since the plugin doesn't work in 5.4 currently, users don't need it now either.